wger/core/views/user.py

```diff
@@ -529,3 +532,62 @@ def get_context_data(self, **kwargs):
                                           _('Gym')],
                                  'users': context['object_list']['members']}
         return context
+
+
+@login_required
+def fitbit_authorisation(request, code=None):
+    template_data = {}
+    client_id = '228GGS'
+    client_secret = 'd651833d76998815030d86edd54c993e'
+
+    fitbit_client = FitbitOauth2Client(client_id, client_secret)
+
+    if 'code'in request.GET:
```

Comment:
The ifs blocks here are deeply nested can you make it more linear for easier read.